Moodlet/app/src/test/java/com/cmput3owo1/moodlet/models/MoodEventTest.java

```diff
@@ -0,0 +1,50 @@
+package com.cmput3owo1.moodlet.models;
+
+import android.graphics.Bitmap;
+
+import org.junit.Test;
+import org.mockito.Mock;
+
+import static org.junit.Assert.*;
+
+public class MoodEventTest {
+
+    @Mock
+    Bitmap bitmap;
+
+    @Test
+    public void testConstructor() {
+        MoodEvent newMoodEvent = new MoodEvent(EmotionalState.EXCITED);
+        assertNotNull(newMoodEvent);
```

Comment:
Same thing here, assert that the constructor sets the proper emotional state

Reply:
Added constructor content validation tests